File: Bubble Sort/Bubble_Sort.java
public class Bubble_Sort{
	public static void main(String...args){
		//array
		int[] A = {0,9,8,7,6,5,4,3,2,1};
		int swap = 0;

		for(int i = 0; i < A.length; i++){
			for(int j = 0; j < A.length - 1; j++){
				if(A[j] > A[j+1]){
					swap = A[j];
					A[j] = A[j+1];
					A[j+1] = swap;
				}
			}
		}
		for(int k = 0; k < A.length; k++){
			System.out.print(A[k] + " ");
		}
	}
}

File: Bubble_Sort.py
# Bubble Sort implementation in Python
"""
ALGORITHM
Step 1: Traverse the array 1st time.
Step 2: Traverse the array 2nd time and check if the current element i.e.(un_sorted[j]) is
        less than previous one i.e. (un_sorted[j-1])if yes then swap.
"""
un_sorted = [0, 4, 2, 3, 5, 1, 3, 5, 6, 3, 5, 6, 2, 4, 6, 3, 5]

for i in range(0, un_sorted.__len__()):
    for j in range(1, un_sorted.__len__() - i):
        if un_sorted[j] < un_sorted[j - 1]:
            temp = un_sorted[j]
            un_sorted[j] = un_sorted[j - 1]
            un_sorted[j - 1] = temp

print(un_sorted)
